rest_framework/schemas/openapi.py

```diff
@@ -57,11 +60,16 @@ def get_schema(self, request=None, public=False):
             paths.setdefault(path, {})
             paths[path][method.lower()] = operation
 
+            if view.__class__.__name__ not in processed_views_for_tags:
+                tags.extend(view.schema.get_tag_objects())
+                processed_views_for_tags.add(view.__class__.__name__)
```

Comment:
@carltongibson @kevin-brown I have a doubt.
Will this logic work for `WrappedAPIView`?
For all functional-based views, `WrappedAPIView` will be the class. 
While processing the first functional-based view, If I add `WrappedAPIView` class to set, remaining all functional-based views will not proceed for the global tag objects.